include/finufft/finufft_core.h

```diff
@@ -160,10 +160,10 @@ template<typename TF> struct FINUFFT_PLAN_T { // the main plan class, fully C++
 
   std::array<UBIGINT, 3> mstu; // number of modes in x/y/z dir (historical CMCL name) =
                                // N1/N2/N3
-  UBIGINT N;                   // total # modes (prod of above three)
+  inline UBIGINT N() { return mstu[0]*mstu[1]*mstu[2]; } // total # modes (prod of above three)
```

Comment:
const?

Reply:
Absolutely correct, thanks for spotting!

BTW, should I do the analogous thing for `nf`?

Reply:
Yes, thanks for spotting